### PR TITLE
Removes The Wall™ from the GTR

### PR DESCRIPTION
**Pull request by sheeplets**
Second PR for me, let's see how much damage I can do this time.

- rscdel: "Removed the wall that was left smack dab in the middle of the General Treatment Room to house the GTR's APC and intercom unit. The intercom is gone as well."
 - tweak: "Relocated one of the General Treatment Room's cryo pods into the Intensive Care Unit to make room for the GTR's APC, along with an extra table."

GTR
ICU

### CODE REVIEW

**Comment by 1Sparky1** [comment]
Personally not a huge fan. I like that wall, as it breaks up the GTR a little, and I've never had any issues navigating around it. Also not certain how I feel about losing a cryogenics pod to the ICU - It's not a major change given that it's very rare to have two people in cryo treatment at once, but one I'd rather not see. However, I'm just one person, so maybe that's not the consensus.

**Comment by sheeplets** [comment]
> Personally not a huge fan. I like that wall, as it breaks up the GTR a little, and I've never had any issues navigating around it. Also not certain how I feel about losing a cryogenics pod to the ICU - It's not a major change given that it's very rare to have two people in cryo treatment at once, but one I'd rather not see. However, I'm just one person, so maybe that's not the consensus.

Just walk into the other room lol

File: html/changelogs/small_medbay_tweaks.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+ - rscdel: "Removed the wall that was left smack dab in the middle of the General Treatment Room to house the GTR's APC and intercom unit. The intercom is gone as well."
```
Comment (fernerr) [inline html/changelogs/small_medbay_tweaks.yml:41]:
```suggestion
 - maptweak: "Removed the wall that was left smack dab in the middle of the General Treatment Room to house the GTR's APC and intercom unit. The intercom is gone as well."
```

**Comment by sheeplets** [comment]
!review

**Review by fernerr — APPROVED** [review]
A long time coming change, the gtr oftentimes becomes a clusterfuck so this'll for sure help alleviate the traffic congestion a bit.

**Review by NonQueueingMatt — APPROVED** [review]

**Comment by FayeLovesU** [comment]
In response to the 'I don't want to use a cryopod',- how often do we actually use them? Moreover, when we do, how often do we use two at he same time? It's fine imo

**Pull request merged**